router/peers.go

```diff
@@ -57,6 +57,15 @@ func (peers *Peers) ForEach(fun func(PeerName, *Peer)) {
 	}
 }
 
+// Create an "improved" update from an incoming update:
```

Comment:
That only half-describes what this function does, i.e. it describes what goes in the result but not what happens to the state.